### ISSUE TITLE
Help running blackbird! 

### ISSUE THREAD

**Issue by AhmedRejeb** [butor/blackbird#531]
Hello,

I'm trying to run blackbird on ubuntu system as described in the readme part but i get this issue in the image below

![image](https://user-images.githubusercontent.com/23346805/112737480-2b871100-8f5b-11eb-966e-1a71caeb3631.png)

any idea what can cause this, thanks in advance

**Comment by sdess09** [comment]
Yeah I had issues with exmo but seeing that it wasn't an exchange that I used I removed it.

**Comment by CptHowdy2012** [comment]
im totally new to cpp, please forgive me. @sdess09 what exactly did you remove to omit the exmo exchange? I tried a few different ways with no luck. I am anxious to get this working. @AhmedRejeb have you managed to get this working?

**Comment by JoseMiguel** [comment]
Use ubuntu 18.04

**Comment by CptHowdy2012** [comment]
just tried with a fresh install of 18.04 desktop and the same result with exmo.cpp

[https://prnt.sc/12bqn6r](url)

**Comment by CptHowdy2012** [comment]
Update: Ubuntu 16.04 Desktop was successful
